src/modules/SystemSettings.rb

```diff
@@ -128,8 +130,13 @@ def Activate
         # set the scheduler
         Bootloader.modify_kernel_params("elevator" => new_elevator)
 
-        # TODO FIXME: set the scheduler for all disk devices,
-        # reboot is required to activate the new scheduler now
+        # activate the scheduler for all disk devices
+        if new_elevator != :missing
+          Dir["/sys/block/*/queue/scheduler"].each do |f|
```

Comment:
ah, so it use bind feature. Will it be persistent as it looks for me like it ends when installation finish as it use insts-sys kernel and not target one.